jbpm-workitems/jbpm-workitems-archetype/src/main/resources/archetype-resources/jbpm-workitems-__rootArtifactId__/src/main/java/__classPrefix__WorkItemHandler.java

```diff
@@ -3,13 +3,31 @@
 import org.jbpm.process.workitem.core.AbstractLogOrThrowWorkItemHandler;
 import org.kie.api.runtime.process.WorkItem;
 import org.kie.api.runtime.process.WorkItemManager;
+import org.jbpm.process.workitem.core.util.Wid;
+import org.jbpm.process.workitem.core.util.WidParameter;
+import org.jbpm.process.workitem.core.util.WidMavenDepends;
 
+@Wid(widfile="${classPrefix}Definitions.wid", name="${classPrefix}Definitions",
+        displayName="${classPrefix}Definitions", icon="${classPrefix}Icon.png",
+        defaultHandler="mvel: new ${package}.${classPrefix}WorkItemHandler()",
+        parameters={
+            @WidParameter(name="sampleParam"),
+            @WidParameter(name="sampleParamTwo")
+        },
+        mavenDepends={
+            @WidMavenDepends(group="${groupId}", artifact="${rootArtifactId}", version="${version}")
+        })
 public class ${classPrefix}WorkItemHandler extends AbstractLogOrThrowWorkItemHandler {
+
     public void executeWorkItem(WorkItem workItem,
                                 WorkItemManager manager) {
         try {
 
-            // handler impl here
+            // sample parameters
+            String sampleParam = (String) workItem.getParameter("SampleParam");
+            String sampleParamTwo = (String) workItem.getParameter("SampleParamTwo");
```

Comment:
Here you have a mismatch with names given in the annotations - here is upper case